packages/core/src/js/tracing/span.ts

```diff
@@ -154,3 +154,33 @@ export function addDefaultOpForSpanFrom(client: Client): void {
     }
   });
 }
+
+export const SPAN_THREAD_ID = 'thread.id';
+export const SPAN_THREAD_ID_MAIN = 1;
+export const SPAN_THREAD_ID_JAVASCRIPT = 2;
```

Comment:
Same as for main I think it's better not to send the ID than hard code 2.

Reply:
Updated with 91632023dfd92652a1d2391741901ecadf4a12c0